modules/components/src/AdvancedFacetView/index.js

```diff
@@ -89,13 +89,13 @@ export default class AdvancedFacetView extends React.Component {
     }
   }
 
-  setSearchTerm = debounce(
-    value =>
-      this.setState({
-        searchTerm: value,
-      }),
-    500,
-  );
+  setSearchTerm = debounce(value => {
+    const { onFilterChange = () => {} } = this.props;
```

Comment:
we have conditional chaining in arranger, so could do `onFilterChange?.(value)` without having to set the default